src/content/en/updates/2017/04/headless-chrome.md

```diff
@@ -0,0 +1,369 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Getting started with Headless Chrome
+
+{# wf_updated_on: 2017-04-27 #}
+{# wf_published_on: 2017-04-27 #}
+{# wf_tags: chrome59,headless,testing #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+{# wf_featured_snippet: Learn about Headless Chrome, shipping in Chrome 59. #}
+
+# Getting started with Headless Chrome {: .page-title }
+
+{% include "web/_shared/contributors/ericbidelman.html" %}
+
+<style>
+figure {
+  text-align: center;
+}
+</style>
+
+### TL;DR {: #tldr .hide-from-toc}
+
+[Headless Chrome](https://chromium.googlesource.com/chromium/src/+/lkgr/headless/README.md)
+is a way to run the Chromium browser in a headless environment. Essentially, running Chrome without chrome! It brings **all modern web platform features** provided
+by Chromium and the Blink rendering engine to the command line.
+
+Why is that useful?
+
+A headless browser is great tool for automated testing and server environments where you
+don't need a visible UI shell. For example, you may want to run some tests against
+a real web page, create a PDF of it, or just inspect how the browser renders an URL.
+
+Headless mode is available on Mac and Linux in Chrome 59. Windows support
+is coming soon!
+{: .note }
+
+## Starting Headless (CLI) {: #cli }
+
+The easiest way to get started with headless mode is to open the Chrome binary
+from the command line. If you've got Chrome 59+ installed, start Chrome with the `--headless` flag:
+
+    chrome \
+      --headless \                   # Runs Chrome in headless mode.
+      --disable-gpu \                # Temporarily needed for now.
+      --remote-debugging-port=9222 \
+      https://www.chromestatus.com   # URL to open. Defaults to about:blank.
+
+**Note**: Right now, you'll also want to include the `--disable-gpu` flag.
+That will eventually go away.
+{: .note }
+
+`chrome` should point to your installation of Chrome. The exact location will
+vary from platform to platform. Since I'm on Mac, I created convenient aliases
+for each version of Chrome that I have installed:
+
+    alias chrome="/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome"
+    alias chrome-canary="/Applications/Google\ Chrome\ Canary.app/Contents/MacOS/Google\ Chrome\ Canary"
+    alias chromium="/Applications/Chromium.app/Contents/MacOS/Chromium"
+
+## Command line features {: features }
+
+In some cases, you may not need to [programmatically script](#node) Headless Chrome.
+There are some [useful command line flags](https://cs.chromium.org/chromium/src/headless/app/headless_shell_switches.cc)
+to perform common tasks.
+
+**Note**: You may also need to include the `--disable-gpu` flag for now when
+running these commands
+{: .note }
+
+### Printing the DOM {: dom }
```

Comment:
You could probably move these sections to a single "Useful flags" section and then just list them all in a table... would be faster to digest